samples/bluetooth/bap_unicast_server/src/main.c

```diff
@@ -657,7 +657,8 @@ static int set_location(void)
 
 	if (IS_ENABLED(CONFIG_BT_PAC_SNK_LOC)) {
 		err = bt_pacs_set_location(BT_AUDIO_DIR_SINK,
-					   BT_AUDIO_LOCATION_FRONT_CENTER);
+						(BT_AUDIO_LOCATION_FRONT_LEFT |
+						BT_AUDIO_LOCATION_FRONT_RIGHT));
```

Comment:
I assume this means that the Pixel phone expect peripheral devices to either be a coordinated set, or support stereo? 

Can you elaborate on this in the commit message and/or add a comment here?